packages/apm/7.16.1/manifest.yml

```diff
@@ -0,0 +1,149 @@
+format_version: 1.0.0
+name: apm
+title: Elastic APM
+version: 7.16.1
+license: basic
+description: Ingest APM data
+type: integration
+categories: ["elastic_stack", "monitoring"]
+release: ga
+conditions:
+  kibana.version: "^7.16.1"
```

Comment:
I was wondering if we should set this to `7.16.0` or `7.16.1`? Given that this is a patch version, I don't see any harm in allowing the patched version for 7.16.0. Any thoughts?

Reply:
Additionally, the APM UI migration in `7.16.1` is pointing to `7.16.0` package. If users manage to install `7.16.1` before this, it might cause problems.

Reply:
Sorry I missed your questions. I think your second comment is a good reason to set a constraint on 7.16.1. Ideally the UI would have been updated to specify 7.16.0, but I guess it's too late for that.